front-endd/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,14 @@
+// added routes
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
-
-const routes: Routes = [];
+import { StudendcrudComponent } from './studendcrud/studendcrud.component';
+const routes: Routes = [
```

Comment:
why do all the routes redirect to the same component? Shall we have different sets of components for different sets of actions?

Like, for 

- `/` welcome component
- '/new' New form componeny
like that